kotlinx-coroutines-test/common/src/TestScope.kt

```diff
@@ -46,6 +47,43 @@ public sealed interface TestScope : CoroutineScope {
      */
     @ExperimentalCoroutinesApi
     public val testScheduler: TestCoroutineScheduler
+
+    /**
+     * A scope for background work.
+     *
+     * This scope is automatically cancelled when the test finishes.
```

Comment:
will `runTest` await until `backgroundScope` finished after cancelling? Or it will just cancel it and complete test leaving `backgroundScope` cancelling in background?

Reply:
It will await the background scope. There's no "background", that is, no separate thread for this.